packages/manager/modules/pci/src/projects/project/index.scss

```diff
@@ -1,4 +1,11 @@
 .pci-project-content {
+  .promotion-value {
+    font-size: 1.3rem;
+  }
+  .cta-sm button {
+    width: 100%;
+  }
```

Comment:
Have you tried the `block` property on the `oui-button` ?
https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-buttons-webcomponent-primary--block